File: A2_q2.cxx
#include<iostream>
using namespace std;

int main(){
float A[3]={7.2, 3, 5};
float B[3]={3,-4,6};
cout<<"A=(7.2, 3, 5)"<<endl;
cout<<"B=(3,-4,6)"<<endl;

//Summation of 2 vector(A+B)
cout<<"A+B = (";
for(int j=0; j<2; j++){
	cout<<A[j]+B[j]<<", ";
}cout<<A[2]+B[2]<<")" <<endl;

//Dot product
float sum=0.0;
for(int i=0; i<3; i++){
	sum=sum+A[i]*B[i];
}
cout<<"Dot product of A and B(A.B) is "<<sum<<"."<<endl;

return 0;
}

File: A2_q2.py
A=[2, 3.4, 5]
B=[-6, 2, 4]

print('A = ',A)
print('B = ', B)

#Summation of two vector
C=[1,1,1]
for i in range(0,3,1):
	C[i]=A[i]+B[i]
print('A+B = ', C)

#Dot product
dot = 0
for j in range(0,3,1):
	dot += A[j]*B[j]
print('A.B = ',dot)

